kitsune/customercare/zendesk.py

```diff
@@ -47,7 +49,6 @@ def _user_to_zendesk_user(self, user):
         return ZendeskUser(
             id=int(id_str) if id_str else None,
             verified=True,
-            email=user.email,
```

Comment:
This will break the `create_user` method below for existing users, because zendesk matches on email in the `create_or_update` method. We need to do something like:
```python
def _user_to_zendesk_user(self, user, include_email=True):

    ...

    return ZendeskUser(

        ...

        **({ "email": user.email } if include_email else {})
    )

...

def update_user(self, user):

    ...

    zendesk_user = self._user_to_zendesk_user(user, include_email=False)

    ...

```